sml_builder/method.py

```diff
@@ -5,39 +5,50 @@
 from sml_builder import app
 from .utils import _page_not_found
 
-STATUS_CLASS = {
-    "In development": "pending",
-    "Complete": "success",
-    "Approved for development": "dead",
-}
-
 
 @app.route("/method/<method>")
 def display_method(method):
```

Comment:
Shouldn't this function be deleted now? As we got the display_method_summary function to route and render the summary pages for the methods in ready to use and future methods.

Reply:
done well spotted